app_server/views/index.pug

```diff
@@ -32,35 +32,52 @@ block content
                         button.btn.btn-lg.btn-default#longBtn
                             | Long term
     .spacer-lg
+    .row
+      .col-sm-12
+        .panel.panel-default.panel-spaced
+          h2.Text-Style-2.remove-line-spc
+            | CLOSE TO YOU.
+          h2.Text-Style-16
+            | Nearest crisis accommodation services.
     .row
         each location in locations
           if location
             .col-lg-4.col-md-12
                 .panel.panel-default.panel-spaced
-                    .panel-heading
-                        a(href='/location/'+location.uri, style='text-decoration: none; color: inherit;')
-                            h2.panel-title= location.name
-                    .row
-                        .col-lg-12.col-sm-5
-                            a(href='location/'+location.uri)
-                              if images[locations.indexOf(location)] != null
-                                img.img-responsive.img-square(src=images[locations.indexOf(location)])
-                              else
-                                img.img-responsive.img-square(src='images/placeholder.png')
-                        .col-lg-12.col-sm-4
-                            .panel-body.text-justify
-                                | #{location.description}
-                        .col-lg-12.col-sm-3
-                            .panel-body
-                                if location.available
-                                    span.text-success
-                                        span.glyphicon.glyphicon-ok
-                                        | &nbsp;Avaliable
-                                else
-                                    span.text-danger
-                                        span.glyphicon.glyphicon-remove
-                                        | &nbsp;Not Avaliable
-                            .panel-body
-                                | Contact: #{location.phoneNumber}
+                    a(href='location/'+location.uri)
```

Comment:
As mentioned below the css names could maybe be named a bit better,

Reply:
Have changes these, can't really name the line spacing styles since each is really specific.

Reply:
In that case if you really think they are specific to that element i would actually inline those styles but anyway thats just my preference. I am sure a bunch of people will jump on me saying thats unmaintainable and it slows it down and there is no caching etc